core/tests/puppeteer-acceptance-tests/spec/exploration-editor-tests/load-complete-and-restart-exploration.spec.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright 2023 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use explorationEditor file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Acceptance Test for preview tab in exploration editor.
+ */
+import {UserFactory} from '../../puppeteer-testing-utilities/user-factory';
+import testConstants from '../../puppeteer-testing-utilities/test-constants';
+import {ExplorationEditor} from '../../user-utilities/exploration-editor-utils';
+
+const DEFAULT_SPEC_TIMEOUT: number = testConstants.DEFAULT_SPEC_TIMEOUT;
+const INTRODUCTION_CARD_CONTENT: string =
+  'Test exploration to question negative numbers';
+const NEW_CARD: string = '/';
+enum INTERACTION_TYPES {
+  CONTINUE_BUTTON = ' Continue Button ',
```

Comment:
Why are there spaces around the texts?

Reply:
Actually, I was using the exact format of the interaction names as they appear in the UI to click on them, which included spaces. This was reflected in the enum values. However, to keep the enum clean and reusable, I've now moved this formatting directly into the function addInteraction where these enum values are used. I've also added a comment in the function to explain this adjustment.